spec/features/tour_list/recordings/index_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe 'tour list recordings', :vcr, type: :feature do
+
+  let(:user) { create(:user) }
+  let(:landmark) { create(:landmark) }
+
+  context 'as a visitor' do
+    scenario 'unable to see tour list landmarks recordings' do
+      visit tour_list_landmark_recordings_path(landmark)
+
+      expect(page.status_code).to eq(404)
+    end
+  end
+
+  context 'as a user' do
+    before :each do
+      recording = create(:recording, landmark: landmark)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+
+      visit landmark_path(landmark)
+      click_button 'Add to tour list'
+
+      visit tour_list_path
+    end
+
+    scenario 'can select existing recording' do
+      expect(page).to have_button('Choose Existing Recording')
+
+      within("#landmark-#{landmark.id}") do
+        click_on 'Choose Existing Recording'
+      end
+
+      expect(current_path).to eq(tour_list_landmark_recordings_path(landmark))
+
+      within(first('.recording-list')) do
+        click_on 'Pick Recording'
+      end
+
+      expect(current_path).to eq(tour_list_path)
+      expect(page).to have_content('The recording has been saved.')
+    end
+  end
+
+  context 'as an admin'
+
+  context 'edge case' do
+    scenario 'user can delete recording that was selected' do
+      recording = create(:recording, landmark: landmark)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+      visit landmark_path(landmark)
+      click_button 'Add to tour list'
+      visit tour_list_path
+
+      within("#landmark-#{landmark.id}") do
+        click_on 'Choose Existing Recording'
+      end
+
+      within(first('.recording-list')) do
+        click_on 'Pick Recording'
+      end
+
+      click_on 'Delete Recording'
```

Comment:
I'm still confused about what's happening in this workflow.

Reply:
The user has the ability to change a recording that they selected for a landmark. In this workflow, the user adds a recording, then user changes their mind about that recording, then user deletes the selected recording, then user has ability to add another recording (not shown in this test).

We can change the wording of the flash messages to avoid confusion.